src/avram/model.cr

```diff
@@ -8,82 +8,27 @@ abstract class Avram::Model
   include Avram::Polymorphic
   include Avram::SchemaEnforcer
 
-  abstract def id
```

Comment:
Do you think we should move this to the `table` macro? Then if you call `table`, you have to have the `id` method defined.

Reply:
At that point it isn't an expected method on `Avram::Model` it's just a method we define for models connected to tables which is what we already do. I don't think it's valuable to keep

Reply:
ok 👍